packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -142,6 +142,9 @@ Future<String> getCodeSigningIdentityDevelopmentTeam(BuildableIOSApp iosApp) asy
       ..close();
 
   final String opensslOutput = await UTF8.decodeStream(opensslProcess.stdout);
+  // Fire and forget dump of the stderr stream so we don't hold onto resources.
+  // Don't care about the result.
+  // ignore: unawaited_futures
```

Comment:
"dump" implies that it goes somewhere.
Don't we want to wait for this, the opensslOutput, and the opensslProcess.exitCode together?